.circleci/config.yml

```diff
@@ -44,6 +44,81 @@ jobs:
           working_directory: ./web
           command: yarn test:unit
 
+  test_e2e:
+    docker:
+      - image: circleci/python:3.6-node-browsers
+      - image: viime/opencpu-base
+        command: |
+          /bin/sh -c "chmod a+w /usr/local/lib/R/site-library && service cron start && /usr/lib/rstudio-server/bin/rserver && apachectl -DFOREGROUND"
+    steps:
+      - checkout
+      # - restore_cache:
+      #     name: Restore Yarn Package Cache
+      #     keys:
+      #       - yarn-packages-{{ checksum "e2e-test/package.json" }}
+      - run:
+          name: Create a virtualenv for installation
+          command: |
+            python3 -m venv install_env
+            . install_env/bin/activate
+      - run:
+          name: Upgrade pip
+          command: pip install -U pip
+      - run:
+          name: Install python
+          command: |
+            pip install .
+            pip install sentry-sdk[flask]
+            pip install --upgrade sentry-sdk
+      # flask run
+      - run:
+          working_directory: ./viime
+          name: Start the development server
+          command: flask run
+          background: true
+      # run docker
+      # - run:
+      #     working_directory: ./devops
+      #     name: Run docker
+      #     command: |
+      #         docker build -t viime .
+      #         docker run -it --rm -p 8004:8004 viime
+      #     background: true
+      # serve the front end
+      - run:
+          working_directory: ./web
+          name: Install dependencies
+          command: yarn
+          background: true
```

Comment:
This shouldn't be a background task. Background tasks are only for cases where you are serving something and need it to be running simultaneously with something else. It's important that the dependencies are installed before you move on to the next step.